cmd/main.go

```diff
@@ -60,6 +62,7 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 	utilruntime.Must(enterpriseApi.AddToScheme(scheme))
 	utilruntime.Must(enterpriseApiV3.AddToScheme(scheme))
+	utilruntime.Must(enterprisev4.AddToScheme(scheme))
```

Comment:
we already have `utilruntime.Must(enterpriseApi.AddToScheme(scheme))` which points to v4